README.md

```diff
@@ -91,4 +91,38 @@ You can then visit http://localhost:5000/ in your web browser to view the app.
 ### Other useful commands
 * `vagrant provision` - Runs any VM provisioning steps specified in the Vagrantfile. Provisioning steps are one-off operations that adjust the system provided by the box.
 * `vagrant suspend` - Suspends any running VM. The VM will be restarted on the next vagrant up command.
-* `vagrant destroy` - Destroys the VM. It will be fully recreated the next time you run vagrant up.
\ No newline at end of file
+* `vagrant destroy` - Destroys the VM. It will be fully recreated the next time you run vagrant up.
+
+### Using Docker containers
+
+Download: 
+* Docker - you'll need to install [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+#### Running the Dev & Prod Container
+
+There are two flavours of containers that can be built, a development and a production image.
+
+The dev container:
+* Uses Flask to run the application
+* Enables Flask's debugging/developer mode to provide detailed logging and feedback.
+* Allows rapid changes to code files without having to rebuild the image each time.
+
+The production container:
+* Uses Gunicorn to run the application
+
+You can create either a development or production image from the same Dockerfile, by running the following:
+```bash
+$ docker build --target development --tag todo-app:dev .
+```
+or the the following for prod:
+```bash
+$ docker build --target production --tag todo-app:prod .
+```
+You can then start the dev container by running:
+```bash
+$ docker run --env-file .env -p 5000:5000 -v $(pwd)/todo_app:/todo_app/app  todo-app:dev
```

Comment:
Slight mistake means the bind mount isn't really being used because you're sharing it in the wrong place in the container.
```suggestion
$ docker run --env-file .env -p 5000:5000 -v $(pwd)/todo_app:/todo_app/todo_app todo-app:dev
```

You should be able to test whether it's working if you:
- Start the container
- Modify index.html in a visible way
- Refresh the page

The change should be visible without having to rebuild the image or even restart the container.

Reply:
Yea, it worked first time, so was doing a negative manual test and copied the wrong command..thanks, corrected.